auto3dseg/notebooks/auto3dseg_autorunner_ref_api.ipynb

```diff
@@ -24,7 +24,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">you don't need training params unless you need it
</pre><p>this sentence is incomplete?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Yes it is a bad sentence.

-> The training params are optional. If you don't need it, you can use either `train()` or `train({})`